peer/pendingheap/heap_test.go

```diff
@@ -55,7 +56,7 @@ func TestPeerHeapOrdering(t *testing.T) {
 	}
 
 	for _, tt := range tests {
-		var h pendingHeap
+		h := pendingHeap{Rand: rand.New(rand.NewSource(0))}
```

Comment:
You could use Source directly if you just need a random integer. The rand object is a thin wrapper.